### DISCUSSION TITLE
How to get IPC_.. Codes?

### DISCUSSION THREAD

**Discussion by mdkumargce** [roleoroleo/yi-hack-Allwinner#306] [Q&A] [answered]
Hi roleoroleo,

Can you tell me how to find these IPC codes which are defined ipc_cmd.h? 

I de-compiled the cloudAPI of my camera(Realtek-Mips arch) but I couldn't find similar hex codes there. I did find few hex strings but they are way too long. Can you please let me know where and how did you get these codes?

Thanks,
Dinesh

**Comment by roleoroleo** [comment, marked as answer]
Stop ipc_multiplexer and sniff ipc message queue with this program:
[ipc_sniff.tar.gz](https://github.com/roleoroleo/yi-hack-Allwinner/files/6520933/ipc_sniff.tar.gz)

**Reply by mdkumargce** [reply]
Thanks. Can you share the source code of ipc_sniff? looks like the binary was compiled for arm-v7 but mine is MIPS-I.

**Reply by roleoroleo** [reply]
Here it is:
[ipc_sniff.tar.gz](https://github.com/roleoroleo/yi-hack-Allwinner/files/6522606/ipc_sniff.tar.gz)
